<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1404</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoteSonne</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rotesonne.qrc">
    <normaloff>:/images/images/media-podcast.png</normaloff>:/images/images/media-podcast.png</iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>241</width>
          <height>100</height>
         </size>
        </property>
        <widget class="QPushButton" name="prevButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../rotesonne.qrc">
           <normaloff>:/images/images/player_rew.png</normaloff>:/images/images/player_rew.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
        <widget class="QPushButton" name="playPauseButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>20</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../rotesonne.qrc">
           <normaloff>:/images/images/player_play.png</normaloff>:/images/images/player_play.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="nextButton">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>61</width>
           <height>61</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../rotesonne.qrc">
           <normaloff>:/images/images/player_fwd.png</normaloff>:/images/images/player_fwd.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>48</width>
           <height>48</height>
          </size>
         </property>
        </widget>
        <widget class="QSlider" name="playerSlider">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>40</y>
           <width>714</width>
           <height>19</height>
          </rect>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Player</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="cover">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>250</width>
              <height>250</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../rotesonne.qrc">:/images/images/nocover.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="title">
                <string>Track information</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
               <property name="checkable">
                <bool>false</bool>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QGridLayout" name="gridLayout" columnstretch="2,1">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="text">
                     <string>Sample rate:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLabel" name="sampleRateInfoLabel">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Channels:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLabel" name="channelsInfoLabel">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>Bit per sample:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLabel" name="bpsInfoLabel">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>Bitrate:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLabel" name="bitrateInfoLabel">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="0">
                   <widget class="QLabel" name="label_6">
                    <property name="text">
                     <string>Total time:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="4" column="1">
                   <widget class="QLabel" name="totalTimeInfoLabel">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>File size:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1">
                   <widget class="QLabel" name="fileSizeInfoLabel">
                    <property name="text">
                     <string/>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,1">
            <item>
             <widget class="QListWidget" name="artistList"/>
            </item>
            <item>
             <widget class="QListWidget" name="albumList"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="trackList">
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tags</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1404</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionCollection"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionCollection">
   <property name="text">
    <string>&amp;Collection</string>
   </property>
   <property name="shortcut">
    <string extracomment="Show collection window">Ctrl+C</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rotesonne.qrc"/>
  <include location="../rotesonne.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
